eventlet/support/afinet.py

```diff
@@ -0,0 +1,46 @@
+"""
+Test availabilty of AF_INET6 on a machine
+
+Uses socket API to determine configuration
+"""
+
+from socket import *
+
+def check_ipv6_lo_addr():
+    try:
+        sock = socket(AF_INET6, SOCK_STREAM)
+        sock.bind(('::1', 0))
+        return True
+    except:
+        return False
+
+def check_ipv6_hosts_file():
+    if not check_ipv6_lo_addr():
```

Comment:
One `bind()` to just `import eventlet` is already too much. Executing this function twice is excessive.